crates/bevy_log/src/once.rs

```diff
@@ -0,0 +1,99 @@
+/// Call [`trace!`] once per call site.
```

Comment:
Btw, I think you need to qualify these links since they're not in scope. (i.e. ``[`trace!`](crate::trace)`` inline or ``[`trace!`]: crate::trace`` at the bottom of the doc comment).